src/components/BaseComponent.vue

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import { vxm } from "@/store";
+
+@Component
+export default class BaseComponent extends Vue {
+  get darkMode(): boolean {
+    return vxm.general.darkMode;
+  }
+
+  get currentUser(): string {
+    return vxm.wallet.currentUser;
```

Comment:
Shall we adopt the new system of not referencing ```vxm``` directly? May as well do this now.

Reply:
sounds good, i will take a look at it

Reply:
done